src/coreclr/gc/gc.cpp

```diff
@@ -31760,8 +31760,8 @@ bool gc_heap::should_sweep_in_plan (heap_segment* region)
         size_t basic_region_size = (size_t)1 << min_segment_size_shr;
         assert (heap_segment_gen_num (region) == heap_segment_plan_gen_num (region));
 
-        int surv_ratio = (int)(((double)heap_segment_survived (region) * 100.0) / (double)basic_region_size);
-        dprintf (2222, ("SSIP: region %p surv %d / %zd = %d%%(%d)",
+        size_t surv_ratio = (size_t)(((double)heap_segment_survived (region) * 100.0) / (double)basic_region_size);
```

Comment:
this is a ratio, it can't be more than 100. there's no need to make this a size_t